src/GitHub/Data/Activities.hs

```diff
@@ -46,13 +46,18 @@ instance FromJSON Subject where
         <*> o .: "type"
 
 data NotificationReason
-    = AssignReason
+    = ApprovalRequestedReason
+    | AssignReason
     | AuthorReason
     | CommentReason
+    | CiActivityReason
```

Comment:
Note that I am not 100% if this is the correct capitalisation, but CIActivity looked fishy. ;-)